lib/js/components/Layout/Layout.stories.ts

```diff
@@ -0,0 +1,94 @@
+import Layout from './Layout.vue';
+
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue3';
+import {
+	LAYOUT_CONTENT_COLUMN_SIZE,
+	LAYOUT_RIGHT_COLUMN_MODE,
+	LAYOUT_RIGHT_COLUMN_SIZE,
+} from './Layout.consts';
+
+export default {
+	title: 'Components/Layout',
+	component: Layout,
+} as Meta<typeof Layout>;
+
+const StoryTemplate: StoryFn<typeof Layout> = (args) => ({
+	components: { Layout },
+	setup() {
+		return { ...args };
+	},
+	template: `
+		<div
+			style='height: 100%; width: 100%; display: flex; min-height: 100vh; border: 1px solid gray; font-size: 12px; color: #fff;'>
```

Comment:
you can add use `layout: 'fullscreen'` to avoid default stroybook paddings in this story